definitions/json/AuthorisationCaseField.json

```diff
@@ -2807,18 +2555,6 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
-  {
-    "CaseTypeId": "ET_EnglandWales",
-    "CaseFieldID": "hearingCollection",
```

Comment:
understand the hearing tab can remain apart from hearing notes. we can set authorisation on the complex type to allow hearingnotes to not be shown and the other fields can be